CONTRIBUTING.md

```diff
@@ -1,27 +1,27 @@
 # Welcome to DocsGPT Contributing Guidelines
 
-Thank you for choosing this project to contribute to. We are all very grateful!
+Thank you for choosing to contribute to our project! We greatly appreciate your help!
 
 ### [🎉 Join the Hacktoberfest with DocsGPT and Earn a Free T-shirt! 🎉](https://github.com/arc53/DocsGPT/blob/main/HACKTOBERFEST.md)
 
 # We accept different types of contributions
 
-📣 Discussions - where you can start a new topic or answer some questions
+📣 Discussions - Engage in conversations, start new topics, or help answer questions.
 
-🐞 Issues - This is how we track tasks, sometimes it is bugs that need fixing, and sometimes it is new features
+🐞 Issues - This is where we keep track of tasks. It could be bug fixes or suggestions for new features.
 
-🛠️ Pull requests - This is how you can suggest changes to our repository, to work on existing issues or add new features
+🛠️ Pull requests - Suggest changes to our repository, either by working on existing issues or adding new features.
 
-📚 Wiki - where we have our documentation
+📚 Wiki - This is where our documentation resides.
 
 
 ## 🐞 Issues and Pull requests
 
-We value contributions to our issues in the form of discussion or suggestions. We recommend that you check out existing issues and our [roadmap](https://github.com/orgs/arc53/projects/2).
+We highly value contributions in the form of discussions or suggestions. We recommend taking a look at existing issues and our [roadmap](https://github.com/orgs/arc53/projects/2).
```

Comment:
Maybe remove the word highly. We do value contributions highly, but it just sounds better without the word